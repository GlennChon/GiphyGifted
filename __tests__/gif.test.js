it("<Gif/> - Expects", () => {
  expect(true).toBeTruthy();
});
