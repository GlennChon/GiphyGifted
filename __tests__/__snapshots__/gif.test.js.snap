// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Gif/> testing component snapshot 1`] = `
<Fragment>
  <div
    className="container-gif"
  >
    <GifPlayerContainer
      gif="fhdUrl"
      still="fhsUrl"
    />
  </div>
</Fragment>
`;
